modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/EditKBArticleDisplayContext.java

```diff
@@ -330,6 +349,20 @@ public String getURLTitlePrefix() throws PortalException {
 		return StringUtil.shorten(sb.toString(), 40) + StringPool.SLASH;
 	}
 
+	public String getUserFormattedDisplayDate() {
+		KBArticle kbArticle = getKBArticle();
+
+		if (kbArticle == null) {
+			return StringPool.BLANK;
+		}
+
+		Format dateFormatDateTime = FastDateFormatFactoryUtil.getDateTime(
+			FastDateFormatConstants.LONG, FastDateFormatConstants.SHORT,
```

Comment:
I suppose the LONG date and SHORT time formats are set because of requirements right?

Reply:
I don't know what those attributes mean, I just copied it from the [init.jsp](https://github.com/ambrinchaudhary/liferay-portal/blob/master/modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/init.jsp#L191-L191) that was currently in use.